test/powershell/Language/Classes/Scripting.Classes.Modules.Tests.ps1

```diff
@@ -71,3 +71,41 @@ Import-Module Random
     }
 
 }
+
+Describe 'Module reloading with Class definition' -Tags "CI" {
+
+    BeforeAll {
+        Set-Content -Path TestDrive:\TestModule.psm1 -Value @'

```

Comment:
The dynamic modules created by `New-Module` behave differently from file-backed modules.  For example, [a dynamic modules' members are usable without importing the module](https://github.com/alx9r/ToolFoundations/blob/e59d34958102cfead9dc18602856277169767787/EnvironmentTests/dynamicModules.Tests.ps1#L41-L44) despite that [the module cannot be retrieved using `Get-Module`](https://github.com/alx9r/ToolFoundations/blob/e59d34958102cfead9dc18602856277169767787/EnvironmentTests/dynamicModules.Tests.ps1#L37-L40) (unless you import it first).  I'm not sure whether the differences are significant in this case, but it seems like to be safe file-backed modules should be used for testing.